### PR TITLE
[DNM]batch allocate timestamp

### PR DESCRIPTION
**Pull request by ngaut**

### CODE REVIEW

**Comment by ngaut** [comment]
PTAL.
Ref: https://github.com/pingcap/pd/pull/183

**Comment by siddontang** [comment]
LGTM

**Comment by ngaut** [comment]
PTAL @qiuyesuifeng

File: proto/pdpb.proto [hunk 1]
```diff
@@ -36,7 +36,8 @@ message Timestamp {
 }
 
 message TsoResponse {
-    repeated Timestamp timestamps = 1;
+    optional uint32 count = 1; 
+    optional Timestamp timestamp = 2;
```
Comment (qiuyesuifeng) [inline proto/pdpb.proto]: Alignment.

**Comment by ngaut** [comment]
PTAL

**Comment by qiuyesuifeng** [comment]
LGTM

**Pull request merged**